packages/ui-utils/src/types.ts

```diff
@@ -117,6 +117,27 @@ export type ToolInvocation =
   | CoreToolCall<string, any>
   | CoreToolResult<string, any, any>;
 
+/**
+ * An attachment that can be sent along with a message.
+ */
+export interface Attachment {
+  /**
+   * The name of the attachment.
+   */
```

Comment:
mention that it's a filename? or even rename to `filename`? or is it more general?

Reply:
done!